specification/devcenter/data-plane/Microsoft.DevCenter/preview/2023-07-01-preview/devbox.json

```diff
@@ -0,0 +1,1967 @@
+          }
+        ],
+        "description": "Lists available pools",
+        "operationId": "DevBoxes_ListPools",
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/PoolListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "devcenter.json#/definitions/CloudError"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "description": "The error code for specific error that occurred.",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "DevBoxes_ListPools": {
+            "$ref": "./examples/Pools_List.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-odata": "#/definitions/Pool"
+      }
+    },
+    "/projects/{projectName}/pools/{poolName}": {
+      "get": {
+        "tags": [
+          "Pools"
+        ],
+        "parameters": [
+          {
+            "$ref": "devcenter.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "devcenter.json#/parameters/ProjectNameParameter"
+          },
+          {
+            "$ref": "#/parameters/PoolNameParameter"
+          }
+        ],
+        "description": "Gets a pool",
+        "operationId": "DevBoxes_GetPool",
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/Pool"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "devcenter.json#/definitions/CloudError"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "description": "The error code for specific error that occurred.",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "DevBoxes_GetPool": {
+            "$ref": "./examples/Pools_Get.json"
+          }
+        }
+      }
+    },
+    "/projects/{projectName}/schedules": {
+      "get": {
+        "tags": [
+          "Schedules"
+        ],
+        "parameters": [
+          {
+            "$ref": "devcenter.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "devcenter.json#/parameters/TopParameter"
+          },
+          {
+            "$ref": "devcenter.json#/parameters/FilterParameter"
+          },
+          {
+            "$ref": "devcenter.json#/parameters/ProjectNameParameter"
+          }
+        ],
+        "description": "Lists available schedules for all resources within a project.",
```

Comment:
To avoid any future ambiguity with the possibility of user-defined schedules, we could phrase this description as:

```
Lists all schedules configured by your project administrator within a project.
```